platform/kindle/koreader.sh

```diff
@@ -392,6 +400,9 @@ if [ "${STOP_FRAMEWORK}" = "no" ] && [ "${INIT_TYPE}" = "upstart" ]; then
         done
         logmsg "Title bar geometry restored to '$(${KOREADER_DIR}/wmctrl -l -G | grep ":titleBar_ID:" | awk '{print $2,$3,$4,$5,$6}' OFS=',')' (ought to be: '${TITLEBAR_GEOMETRY}') [after ${WMCTRL_COUNT} attempts]"
     fi
+    for job in ${TOGGLED_SERVICES}; do
+        start "${job}"
```

Comment:
Won't it be a problem when `otav3`, `otaupd` are disabled because of OTA reasons? 
As described in: https://www.mobileread.com/forums/showthread.php?t=327879&page=4

Reply:
The command is allowed to fail, so, no?

Reply:
Thanks.